server/submission/definitions.js

```diff
@@ -18,7 +18,7 @@ const typeDefs = `
       uploadManuscript(id: ID!, file: Upload!): Manuscript!
       finishSubmission(data: ManuscriptInput!): Manuscript!
     }
-    
+
```

Comment:
Is the deletion of all these lines coming from your editor's linting? Personally, I find it easier to read with spaces in between, but either way, we should make it consistent throughout this file

Reply:
yes - it removes unnecessary whitespace - not sure what you mean?